blockstack/blockstack-app
# Send users back into the onboarding state they left off on

How to QA this:

- Use the demo app, start the onboarding process
- Close the auth popup while you're on an onboarding screen
  - this can be "view secret key", "save secret key", or "username"
- Back in the demo app, start onboarding again
  - You should go back to the page that you left off on
- Finish auth - it should work as normal
- Sign out of authenticator
- Start onboarding flow again
  - ensure that you go through the flow from the start
